packages/datepickers/src/styled/StyledHeaderPaddle.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+import { retrieveSpacing } from './utils';
+
+const COMPONENT_ID = 'datepickers.header_paddle';
+
+const StyledHeaderPaddle = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID
+})<{
+  isSmall: boolean;
+  isHidden?: boolean;
```

Comment:
Consider the value of connecting with the native global `[hidden]` attribute: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden. YMMV. But the fact that you are specifying the `display` below makes this a viable option.

Reply:
(per offline convo)